Packages/packages-lock.json

```diff
@@ -254,6 +254,15 @@
       },
       "url": "https://packages.unity.com"
     },
+    "com.unity.rendering.hybrid": {
```

Comment:
We've stopped using the hybrid renderer due to the need for skinned meshes. Please revert.